resources/views/volumes/clone.blade.php

```diff
@@ -0,0 +1,195 @@
+@extends('app')
+
+@section('title', 'clone volume')
+
+@push('scripts')
+    <script type="text/javascript">
+        biigle.$declare('destinationProjects', {!!$destinationProjects!!});
+        biigle.$declare('volume', {!!$volume!!});
+        biigle.$declare('name', '{!!old('name',$volume->name)!!}');
+        biigle.$declare('fileLabelTrees', '{!!$labelTrees!!}');
+        biigle.$declare('selectedFilesIds', '{!! collect(old('only_files',[])) !!}');
+        biigle.$declare('selectedFiles', '{!! collect(old('selected_files',[])) !!}');
+        biigle.$declare('fileLabelIds', '{!! collect(old('only_file_labels',[])) !!}');
+        biigle.$declare('annotationLabelTrees', '{!!$labelTrees!!}');
+        biigle.$declare('cloneFileLabels', {{old('clone_file_labels',false)}});
+        biigle.$declare('cloneAnnotations', {{old('clone_annotations',false)}});
+        biigle.$declare('annotationLabelIds', '{!! collect(old('only_annotation_labels', [])) !!}');
+        biigle.$declare('cloneUrlTemplate', "{{ url("api/v1/volumes/{$volume->id}/clone-to/:pid") }}")
+    </script>
+@endpush
+
+@section('content')
+    <div class="container">
+        <div class="col-sm-8 col-sm-offset-2 col-lg-6 col-lg-offset-3">
+            <div class="form-group">
+                <h2 class="row">Clone volume "{{$volume->name}}"</h2>
+                <br>
+            </div>
+            <form id="clone-volume-form" class="clearfix" role="form" method="POST" :action="getCloneUrl"
+                  enctype="multipart/form-data" v-on:submit="startLoading" v-on:keydown.enter.prevent>
```

Comment:
Why this?

```suggestion
                  enctype="multipart/form-data" v-on:submit="startLoading">
```

Reply:
Because otherwise, if you want to search for specific files and hit enter then the form is submitted.

Reply:
I see. This must be fixed in the typeahead component. If you can, delete this attribute of the `<form>` in your next update of the PR and I will take care of the typeahead component later.